src/actions/projectActions.ts

```diff
@@ -20,6 +21,19 @@ export function getProjects(): (dispatch: Dispatch<Action>) => void {
   };
 }
 
+export function getProject(
```

Comment:
lol i have a getProject action as well. will have to do a merge conflict once my stuff is in

Reply:
That should be okay, I'm sure they will be doing the same thing!